docs/administering_lagoon/graphql_api.md

```diff
@@ -426,4 +435,4 @@ mutation {
     }
   )
 }
-```
\ No newline at end of file
+```
```

Comment:
Let's also add an example using `projectByName` which shows the groups assigned to a project and their members & roles too